.github/renovate.json

```diff
@@ -48,6 +48,11 @@
         "react-i18next"
       ],
       "labels": ["i18next"]
+    },
+    {
+      "groupName": "engines",
+      "matchManagers": ["engines"],
```

Comment:
I'm not convinced this is the right matcher, I think matchManagers would be something like `["npm"]`. Unless you have docs that say something else?